testnet/stacks-node/conf/krypton-follower-conf.toml

```diff
@@ -2,6 +2,8 @@
 rpc_bind = "0.0.0.0:20443"
 p2p_bind = "0.0.0.0:20444"
 bootstrap_node = "048dd4f26101715853533dee005f0915375854fd5be73405f679c1917a5d4d16aaaf3c4c0d7a9c132a36b8c5fe1287f07dad8c910174d789eb24bdfb5ae26f5f27@krypton.blockstack.org:20444"
+seed = "d29320a09b6ad4e1ed8ffada2b515505a6d95f4bbd167e98cfaf8ef5ef849f6601"
+miner = true
```

Comment:
This config was meant to be a follower config, do you mind creating another config, for miners?

Reply:
Yup, sorry about that.  Fixed.